integration_test/cloudbuild-integration-test.yaml

```diff
@@ -416,6 +437,24 @@
   waitFor:
   - check-store-curl-destination-2
 
+- name: 'busybox:latest'
+  id: check-store-diff-with-local-backup
+  args:
+    - diff
+    - 'integration_test/downloaded2.dcm'
+    - 'integration_test/data/example-redacted-jp2k.dcm'
```

Comment:
Should be diffed with    integration_test/data/example2.dcm, and example3.dcm for the other step, because as mentioned we do not redact uids here.